traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -1859,3 +1859,36 @@ def static_dns_entries_data_post(to_session: TOSession, request_template_data: l
 	if msg is None:
 		logger.error("static_dns_entries returned by Traffic Ops is missing an 'id' property")
 		pytest.fail("Response from delete request is empty, Failing test_case")
+
+  
+@pytest.fixture(name="server_server_capabilities_post_data")
+def server_server_capabilities_data_post(to_session: TOSession, request_template_data: list[JSONData],
+		server_post_data:dict[str, object], server_capabilities_post_data:dict[str, object]
+		) -> dict[str, object]:
+	"""
+	PyTest Fixture to create POST data for server server capabilities endpoint.
+	:param to_session: Fixture to get Traffic Ops session.
+	:param request_template_data: Fixture to get Server Server Capabilities request template from a prerequisites file.
+	:returns: Sample POST data and the actual API response.
+	"""
+
+	server_server_capabilities = check_template_data(request_template_data["server_server_capabilities"], "server_server_capabilities")
+
+	# Return new post data and post response from server server capabilities POST request
+
+	server_id = server_server_capabilities["serverId"]
+	server_capability = server_server_capabilities["serverCapability"]
+
+	logger.info("New server_server_capabilities data to hit POST method %s", server_server_capabilities)
+
+	# Hitting server server capabilities POST method
+	response: tuple[JSONData, requests.Response] = to_session.associate_server_capability_to_server(server_id=server_id, data=server_server_capabilities)
+	resp_obj = check_template_data(response, "server_server_capabilities")
+	yield resp_obj
+	server_id = resp_obj.get("serverId")
+	msg = to_session.delete_server_capability_association_to_server(query_params=server_id, server_capability=server_capability)
```

Comment:
This won't delete the data, You are passing two arguments query_params and server_capability,
But the delete method in tosession accepts only query_params,
`delete_server_capability_association_to_server(self, query_params=None)`
Just pass the query_params data itself as,
`msg = to_session.delete_server_capability_association_to_server(query_params = {"serverId":server_id, "serverCapability":server_capability})`

Reply:
Yes this is true. I made this change. I'm not sure why it didn't push with the updated code.